lib/scsLib.js

```diff
@@ -52,7 +52,7 @@ class ScsLib {
 	By setting the view to 'provider', when the doc says subscribe, we subscribe.
 	*/
   isProvidererView(info, params) {
-    const view = this.getParamOrDefault(info, params, 'view', 'x-view', 'client');
```

Comment:
you want to make it by default to not follow the spec?

Reply:
Actually, the default here is to follow the spec. But I'll remove that default from this line because I moved them into package.json.

Reply:
Actually, the default here is to follow the spec. But I'll remove that default from this line because I moved them into package.json.